packages/babel-plugin-transform-parameters/test/fixtures/regression/11344-flow/input.js

```diff
@@ -0,0 +1,7 @@
+function a(b: (c) => void = {}) {
+  let c;
+}
+
+function d(e = <T>() => {}) {
```

Comment:
nit: This test will be passing without current change since we are not registering bindings when parsing type parameters.